.github/workflows/issues.yml

```diff
@@ -39,17 +37,14 @@
                       "id": "12"
                   },
                   "customfield_12751": [{
-                          "id": "22222"
-                  }],
+                          "id": "25550"
```

Comment:
My assumption is that this changes jira ticket from my team to APIx Integrations Triage?

Reply:
discussed offline